src/full_node/full_node.py

```diff
@@ -287,6 +293,26 @@ async def _on_connect(self) -> OutboundMessageGenerator:
         async for msg in self._send_tips_to_farmers(Delivery.RESPOND):
             yield msg
 
+    async def periodically_peer_gossip(self):
+        while True:
```

Comment:
How will this task ever exit? Be very judicious in creating new tasks because they love to bite us later.